### PR TITLE
RHEL 8 Support

### PR DESCRIPTION
**Pull request by Loquacity**
https://github.com/SUSE/spacewalk/issues/9574

### CODE REVIEW

**Comment by Loquacity** [comment]
Right now, the RH8 columns are the same as the RH7 columns. They will need to be edited accordingly.

File: modules/client-configuration/pages/supported-features.adoc [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [[supported-features]]
```
Comment (keichwa) [inline modules/client-configuration/pages/supported-features.adoc]:
```suggestion
[[supported-clients]]
```
I did not check whether we reference the label already.

**Comment by mcalmer** [comment]
We write that v8 only support salt, but we use the green checkmark in the feature matrix.
It feel a but confusing. Shouldn't we better use "Salt" everywhere, where we have the checkbox now ?

**Comment by Loquacity** [comment]
> We write that v8 only support salt, but we use the green checkmark in the feature matrix.
> It feel a bit confusing. Shouldn't we better use "Salt" everywhere, where we have the checkbox now ?

Yes. Right now, all the 8 columns are exactly the same as the 7 columns.

**Comment by Loquacity** [comment]
> > We write that v8 only support salt, but we use the green checkmark in the feature matrix.
> > It feel a bit confusing. Shouldn't we better use "Salt" everywhere, where we have the checkbox now ?
> 
> Yes. Right now, all the 8 columns are exactly the same as the 7 columns.

Updated.

**Comment by Loquacity** [comment]
@mcalmer @paususe can you please review so i can get this merged?

File: modules/client-configuration/pages/supported-features-centos.adoc [hunk 2]
```diff
@@ -20,51 +20,48 @@ The icons in this table indicate:
...
+| System custom states | ``Salt`` | ``Salt`` | ``Salt``
+| Group custom states | ``Salt`` | ``Salt`` | ``Salt``
+| Organization custom states    | ``Salt`` | ``Salt`` | ``Salt``
+| System set manager (SSM) | icon:check[role="green"] | icon:check[role="green"] | ``Salt``
+| Service pack migration | icon:times[role="danger"] | icon:times[role="danger"] | icon:times[role="danger"]
+| Basic Virtual Guest Management icon:asterisk[role="none"] | ``Traditional`` | icon:check[role="green"] | ``Salt``
+| Advanced Virtual Guest Management icon:asterisk[role="none"] | icon:times[role="danger"] | ``Salt`` | ``Salt``
+| Virtual Guest Installation (Kickstart), as Host OS | ``Traditional`` | ``Traditional`` | icon:times[role="danger"]
+| Virtual Guest Installation (image template), as Host OS | ``Traditional`` | icon:check[role="green"] | ``Salt``
+| System redeployment ({ay}) | ``Traditional`` | icon:check[role="green"] | ``Salt``
```
Comment (mcalmer) [inline modules/client-configuration/pages/supported-features-centos.adoc]: Not sure if the macro {ay} is here correct. We for redhat it is "kickstart", not "autoyast"

File: modules/client-configuration/pages/supported-features-centos.adoc [hunk 3]
```diff
@@ -20,51 +20,48 @@ The icons in this table indicate:
...
-| Build Docker image with OS | icon:times[role="danger"] | icon:times[role="danger"]
-| Kiwi buildhost |  icon:times[role="danger"] | icon:times[role="danger"]
-| Build Kiwi image with OS |  icon:times[role="danger"] | icon:times[role="danger"]
+``Salt``: ZeroMQ, Salt-SSH | ``Traditional``: OSAD, RHNSD, SSH-push. ``Salt``: ZeroMQ, Salt-SSH | ``Salt``: ZeroMQ, Salt-SSH
+| Works with {productname} Proxy | icon:check[role="green"] | icon:check[role="green"] | ``Salt``
+| Action chains | icon:check[role="green"] | icon:check[role="green"] | ``Salt``
+| Software crash reporting | icon:times[role="danger"] | ``Traditional`` | icon:times[role="danger"]
+| Staging (pre-download of packages) | icon:check[role="green"] | icon:check[role="green"] | ``Salt``
+| Duplicate package reporting | icon:check[role="green"] | icon:check[role="green"] | ``Salt``
+| CVE auditing |  icon:check[role="green"] | icon:check[role="green"] | ``Salt``
```
Comment (mcalmer) [inline modules/client-configuration/pages/supported-features-centos.adoc]: For all CentOS versions: this requires errata data with CVE IDs. (see also "Apply patches")

**Pull request merged**